src/components/app/AppViewRouter.js

```diff
@@ -95,6 +96,7 @@ class AppViewRouterImpl extends PureComponent<AppViewRouterProps> {
   render() {
     return (
       <Fragment>
+        <DowntimeWarning />
         <ServiceWorkerManager />
```

Comment:
I've put the "warning" component before the "service worker manager" component so that the service worker reload notice will be shown over the "warning" component if they're both rendered. I thought it was more important to see the reload notice if the user doesn't close the other warning.
I considered implemented a "notification zone" at the top, so that the notifications would stack instead of cover each other, but decided that it is more work than necessary because the "DowntimeWarning" is here for a few days only.